examples/server/README.md

```diff
@@ -168,7 +168,7 @@ node index.js
 
     `ignore_eos`: Ignore end of stream token and continue generating (default: false).
 
-    `logit_bias`: Modify the likelihood of a token appearing in the generated text completion. For example, use `"logit_bias": [[15043,1.0]]` to increase the likelihood of the token 'Hello', or `"logit_bias": [[15043,-1.0]]` to decrease its likelihood. Setting the value to false, `"logit_bias": [[15043,false]]` ensures that the token `Hello` is never produced (default: []).
+    `logit_bias`: Modify the likelihood of a token appearing in the generated text completion. For example, use `"logit_bias": [[15043,1.0]]` to increase the likelihood of the token 'Hello', or `"logit_bias": [[15043,-1.0]]` to decrease its likelihood. Setting the value to false, `"logit_bias": [[15043,false]]` ensures that the token `Hello` is never produced. The tokens can also be represented as strings, e.g. `[["Hello, World!",false]]` will ban all tokens that represent the string `Hello, World!`. (default: []).
```

Comment:
This seems like a poor example. Banning all of the tokens in "Hello, World!" is going to ban "Hello", comma, "World", and exclamation point from appearing in *any* context.

That sounds like a use case for banning *sequences* of tokens - like how grammar currently restricts the allowed next token, or better, how #2593 handles repetition penalty (with backtracking).

When setting the value to false, I think the most useful tokens to ban are special or prompt-related - EOS, "###", square brackets, and stuff like that.

Reply:
> Banning all of the tokens in "Hello, World!" is going to ban "Hello", comma, "World", and exclamation point from appearing in any context.

That was kind of the point - to ban all individual tokens/words in any context. Banning a sequence was never my intention. But I can see how it can be confusing. I changed the wording to be less ambiguous.